Deploy/deploy-ab2d-to-cms.sh

```diff
@@ -1114,29 +1297,52 @@ if [ -n "${BUILD_NEW_IMAGES}" ]; then
 
   # Note that I can't build with "make docker-build" because test containers
   # try to run inside the docker container. Using "mvn clean package" instead.
-  mvn clean package
+  export OKTA_CLIENT_ID=0oa2t0lsrdZw5uWRx297
+  export OKTA_CLIENT_PASSWORD=HHduWG6LogIvDIQuWgp3Zlo9OYMValTtH5OBcuHw
```

Comment:
You probably don't need these two if you're skipping tests.

Reply:
Thanks, I added "build with tests" and commented them out (in case we later want to make it a deployment option). "Skipping tests" section is active.